strr-web/pages/application-details/[id]/ltsa/index.vue

```diff
@@ -17,10 +17,10 @@
         </div>
       </BcrosBanner>
     </div>
-    <div class="mt-[104px] m:mt-[74px]">
+    <div class="mt-[104px] m:mt-[152px]">
       <div>
         <BcrosTypographyH2
-          class="font-bold mb-6 mobile:mx-2"
+          class="font-bold mobile:mx-2"
```

Comment:
These headers both render as 
```html
<h2 class="undefined font-bold text-[24px] py-[25px] font-bold mobile:mx-2" data-test-id="h2">LTSA Title Owners</h2>
```
Is **undefined** correct?

Reply:
Fixed it.

Reply:
Checked!